cla_public/apps/base/tests/test_healthchecks.py

```diff
@@ -38,7 +38,11 @@ def test_disk_space_check_fails_when_less_than_2_percent_space_is_available(self
 class BackendAPIHealthcheckTest(unittest.TestCase):
     def setUp(self):
         self.app = app.create_app("config/testing.py")
-        self.app.test_request_context().push()
+        self.ctx = self.app.test_request_context()
```

Comment:
Non-blocking, but we seem to be doing exactly the same thing in lots of places (we already were, of course).
I think it would be a nice design to make a subclass of `unittest.TestCase` that implements the `setUp` and `tearDown` methods once, and then inherit from that elsewhere.